src/Integration.Vsix/Commands/HelpMenu/AboutCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.Core;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.Commands
+{
+    internal class AboutCommand : VsCommandBase
+    {
+        internal const int AboutCommandId = 0x104;
+
+        private readonly string aboutUrl = "https://marketplace.visualstudio.com/items?itemName=SonarSource.SonarLintforVisualStudio2022";
```

Comment:
Ah, small problem here, if this is running in VS 2019 then technically the market page is incorrect. I don't have a solution for this, a better way would be to show an About screen and we already have a ticket for this. Let's leave it for now.